src/command.rs

```diff
@@ -282,6 +282,10 @@ fn parse_lengths(body: &[u8]) -> Result<ParsedLengths, FromSliceError> {
         return Ok(parsed);
     }
 
+    if body.len() < 3 {
```

Comment:
Is this not already fixed by https://github.com/ycrypto/iso7816/pull/5 ? See the lines above:

```rust
    } else if l < 3 {
        return Err(FromSliceError::InvalidSliceLength);
    }
```

Reply:
Ah, good point.  I prepared the patch before that PR and didn’t notice that it is already fixed during the rebase.  I will remove that part, but I think the quickcheck tests still make sense.

Reply:
Yes, Its also probably a good idea to copy part of your parsing test and add it to my fuzzing branch